.circleci/config.yml

```diff
@@ -1,5 +1,181 @@
 version: 2.1
 
+orbs:
+  # Using inline orb for now
+  getting-started-smoke-test:
+    orbs:
+      macos: circleci/macos@2
```

Comment:
Should this have a tighter constraint than "2"?

Reply:
changed to use 2.2.0